dev-python/haiku-pyapi/haiku_pyapi-0.3.recipe

```diff
@@ -35,7 +38,7 @@ BUILD_PREREQUIRES="
 	cmd:jam
 	"
 
-PYTHON_VERSIONS=(3.9 3.10)
+PYTHON_VERSIONS=(3.9 3.10 3.11 3.12)
```

Comment:
I don't think the newer versions are intended to be used, maybe @OscarL can shed some more information here?

Reply:
Haiku-PyAPI isn't available in pypi IIRC, so it can't be installed via pip. IMHO in this case it makes sense to provide a package for all supported python versions (maybe even add 3.13?).